src/appengine/handlers/cron/backup.py

```diff
@@ -44,25 +49,33 @@ def get(self):
       return
 
     kinds = [
-        kind.kind_name
-        for kind in metadata.Kind.all()
-        if (not kind.kind_name.startswith('_') and
-            kind.kind_name not in EXCLUDED_MODELS)
+        kind for kind in ndb.Model._kind_map  # pylint: disable=protected-access
```

Comment:
what was wrong with Kind.all ?

we should avoid using internal methods/fields since they may not exist in the python 3 rewrite.

Reply:
Discussed over chat, metadata is coming from appengine sdk, so this dependency needs to be removed. Later when we fix ndb we need to verify this _kind_map works.